src/radical/pilot/utils/component.py

```diff
@@ -314,7 +314,8 @@ def control_cb(self, topic, msg):
         This callback can be overloaded by the component to handle any control
         message which is not already handled by the component base class.
         '''
-        pass
+
+        self._log.debug('ctrl msg ignored: %s:%s', topic, msg.get('cmd'))
```

Comment:
should this also be `debug_4`? (or some other sub-level for debug)

Reply:
ack